src/common/config_opts.h

```diff
@@ -1078,7 +1096,11 @@ OPTION(filestore_max_sync_interval, OPT_DOUBLE, 5)    // seconds
 OPTION(filestore_min_sync_interval, OPT_DOUBLE, .01)  // seconds
 OPTION(filestore_btrfs_snap, OPT_BOOL, true)
 OPTION(filestore_btrfs_clone_range, OPT_BOOL, true)
+#if defined(__linux__)
```

Comment:
@tchaikov @ukernel 
I have not done so, because I would liek to first see that things are working with ZFS as expected.
And then actually start integrating it fully.
Then the options would also become real zfs options instead of:
    `filestore_*_xattr_*_other`
For now I'm just overulling the default on FreeBSD, otherwise there was going to be another lage set of patches for that. But I'll do that in a set of dedicated ZFS patches.